integrated_channels/blackboard/client.py

```diff
@@ -175,6 +224,21 @@ def _validate_course_id(course_id, external_id):
                 HTTPStatus.NOT_FOUND.value
             )
 
+    @staticmethod
+    def _validate_full_course_metadata(channel_metadata_items):
+        """
+        Raise error if course_metadata, course_content_metadata or course_child_content_metadata are invalid.
+        """
+        metadata_set = set(channel_metadata_items.keys())
+        course_metadata_groups = {'course_metadata', 'course_content_metadata', 'course_child_content_metadata'}
+        if not course_metadata_groups.issubset(metadata_set):
+            raise ClientError(
+                'Could not find course metadata group(s):{} necessary to create Blackboard integrated course.'.format(
+                    course_metadata_groups - metadata_set
```

Comment:
wait isn't metadata_set expected to be the larger set in general, is this subtraction reversed if the intent is to show the set difference? or it is better logging wise to show both sets? just curiuos what this output would look like

Reply:
this outputs any of three course metadata objects that are missing like:
`{'course_metadata', 'course_child_content_metadata`}

Reply:
so the log would show you anything that wasn't able to be exported

Reply:
interesting, I maybe was not clear on how the - operator works with these sets... but cool if you have verified it

Reply:
```
>>> set_a = {1, 2, 3}
>>> set_b = {2, 3, 8}
>>> set_c = set_a - set_b
>>> set_c
{1}
```
I have a set of the expected, `you should have these` content items. I subtract what the exporter supplies from that set so any missing should be the remainder.